config/playground/helm/dev/helm-opensearch-dashboards.yaml

```diff
@@ -150,3 +150,8 @@ affinity: {}
 
 # -- Array of extra K8s manifests to deploy
 extraObjects: []
+
+# specify the plugin to install
+plugins:
+  enabled: true
+  installList: ["https://github.com/BionIT/google-analytics-plugin/releases/download/2.4.1/googleAnalytics-2.4.1.zip"]
```

Comment:
should we copy this **zip** file to our repo, create a plugin folder to host all plugins zip file? Thanks

Reply:
I think we can move google analytics repo to the org folder if we want so that we have the source. For artifact, ideally we should have a common place to store them, and I think using an object storage or keeping in a public repository are both fine~ We can discuss more